core-rust/state-manager/src/jni/mempool.rs

```diff
@@ -218,6 +218,15 @@ impl From<PendingTransaction> for JavaRawTransaction {
     }
 }
 
+impl From<NotarizedTransaction> for JavaRawTransaction {
+    fn from(transaction: NotarizedTransaction) -> Self {
+        JavaRawTransaction {
+            payload: manifest_encode(&transaction).unwrap(),
```

Comment:
minor issue, but looks like we're unnecessarily manifest_encode-ing twice (here and in user_payload_hash() below)